oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/EBPFProfilingQueryService.java

```diff
@@ -83,7 +97,17 @@ private EBPFProfilingAnalyzer getProfilingAnalyzer() {
     }
 
     public List<EBPFProfilingSchedule> queryEBPFProfilingSchedules(String taskId, Duration duration) throws IOException {
-        return getScheduleDAO().querySchedules(taskId, duration.getStartTimeBucket(), duration.getEndTimeBucket());
+        final List<EBPFProfilingSchedule> schedules = getScheduleDAO().querySchedules(taskId, duration.getStartTimeBucket(), duration.getEndTimeBucket());
+        if (CollectionUtils.isNotEmpty(schedules)) {
+            schedules.forEach(p -> {
+                try {
+                    p.setProcess(getMetadataQueryDAO().getProcess(p.getProcessId()));
```

Comment:
Please implement multiple get. I am not sure whether process ID is the row ID of the process entity?

Reply:
Yes, the process id is the row ID in storage. I will update to use multiple get.

Reply:
Updated.